ingestion/functions/aws_access/globaldothealth_configuration.py

```diff
@@ -0,0 +1,3 @@
+AWS_REGION = "us-east-1"
+INGESTION_SOURCES_BUCKET = "epid-sources-raw"
+DEFAULT_SCHEDULE_EXPRESSION = "rate(1 day)"
```

Comment:
Might be better to make these values come from environment variables, and keep these values as defaults.

Reply:
+1, config can then be in the GitHub Actions yaml

Reply:
OK, will read from env then. Thanks!